AdyenNetworking/Helpers/Logging.swift

```diff
@@ -22,3 +29,38 @@ internal func adyenPrint(_ items: Any..., separator: String = " ", terminator: S
         idx += 1
     } while idx < endIdx
 }
+
+@_spi(AdyenInternal)
+public func adyenPrintAsJSON(_ dictionary: [String: Any]) {
```

Comment:
is this function is used in other modules?

Reply:
yes, it will be used in the Adyen Components SDK

Reply:
can we simply make it public? it is a bit strange to inroduce use of `@_spi` just for one function

Reply:
printing is not a core functionality of the SDK, its a convenience for other Adyen SDK's, making it just public is even weirder IMO